plat/hisilicon/hikey/hikey_bl31_setup.c

```diff
@@ -152,6 +153,20 @@ void bl31_plat_arch_setup(void)
 			   BL31_COHERENT_RAM_LIMIT);
 }
 
+/* Initialize EDMAC controller with non-secure mode. */
+static void hikey_edma_init(void)
+{
+	int i;
+	uint32_t non_secure;
+
+	non_secure = EDMAC_SEC_CTRL_INTR_SEC | EDMAC_SEC_CTRL_GLOBAL_SEC;
+	mmio_write_32(EDMAC_SEC_CTRL, non_secure);
+
+	for (i = 0; i < EDMAC_CHANNEL_NUMS; i++) {
+		mmio_write_32(EDMAC_AXI_CONF(i), (1 << 6) | (1 << 18));
```

Comment:
For (1 << 6 ) and (1 <<18) - perhaps having these as defines in hi6220.h would be clearer.